app/src/main/res/layout/preference_manual_add_search_engine.xml

```diff
@@ -13,7 +13,8 @@
     <android.support.design.widget.TextInputLayout
         android:id="@+id/edit_engine_name_layout"
         android:layout_width="match_parent"
-        android:layout_height="wrap_content">
+        android:layout_height="wrap_content"
+        android:paddingBottom="@dimen/textinputlayout_spacing">
```

Comment:
nit: why not just hard-code this in the XML document?

I find it's easier to hardcode styles that you don't share:
- You don't have to enter a new file to understand what the layout is doing
- The dimens file stays much more readable

Reply:
Didn't hard-code at @antlam's request - he said we'll need this later so I made it a dimen.